### PR TITLE
Fixing small bug

### PR DESCRIPTION
**Pull request by arnaudon**

### CODE REVIEW

**Comment by arnaudon** [comment]
the change in coverage is very minor, let's discard it

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 default_language_version:
-  python: python3.8
+  python: python3
```
Comment (adrien-berchet) [inline .pre-commit-config.yaml:2]: I'm not sure it's a good idea (I think it's just the same as removing it) but let's try

**Review by adrien-berchet — APPROVED** [review]

File: .pre-commit-config.yaml (hunk 1 above)
Comment (arnaudon) [reply .pre-commit-config.yaml:2]: ah sorry, I can put it back, I now installed python3.8 on my mac

Comment (arnaudon) [reply .pre-commit-config.yaml:2]:
```suggestion
  python: python3.8
```

**Pull request merged**